clients/admin-ui/src/features/common/Layout.tsx

```diff
@@ -1,3 +1,6 @@
+/* istanbul ignore file */
```

Comment:
I will remove these lines as soon as I can figure out the linting issues as documented below.